### PR TITLE
fix(Visual): add SRP support on CameraColorOverlay

### PR DESCRIPTION
**Pull request by Wolar**
Probably related to https://github.com/ExtendRealityLtd/VRTK/issues/1959 so I didn't open a new issue.

### CODE REVIEW

File: Runtime/Visual/CameraColorOverlay.cs [hunk 1]
```diff
@@ -271,6 +296,16 @@ protected virtual void CancelBlinkRoutine()
             }
         }
 
+        /// <summary>
+        /// The moment before <see cref="Camera"/> render that will apply the <see cref="Color"/> overlay.
+        /// </summary>
+        /// <param name="context">Not used, used only to match SRPs endCameraRendering method signature</param>
```
Comment (bddckr) [inline Runtime/Visual/CameraColorOverlay.cs]:
```suggestion
        /// <param name="context">Ignored. Only used to match the method signature of <see cref="RenderPipelineManager.endCameraRendering"/>.</param>
```

**Comment by Wolar** [comment]
No problem with that change, should be changed now

**Comment by bddckr** [comment]
We're friends of the idea of "once a PR is merged the commits it brought should be primarily helpful for the readers of the commits". As such we'd like you to squash the commits before we merge (whether we merge is pending someone else to review). So just squash and force-push! (GitHub thankfully has good UX for reviewing force-pushes even.)

**Comment by Wolar** [comment]
Ok, no problem, done

**Comment by Wolar** [comment]
I just found out that this solution is not enough for Quest. For some reason I'm getting 

OPENGL NATIVE PLUG-IN ERROR: GL_INVALID_OPERATION: Operation illegal in current state 
(Filename: ./Runtime/GfxDevice/opengles/GfxDeviceGLES.cpp Line: 353)

I've tried to replace that GL rendering code in PostRender with something like code bellow for SRP 
`var cmd = CommandBufferPool.Get("FadeScreenPass");
  cmd.SetViewProjectionMatrices(Matrix4x4.identity, Matrix4x4.identity);
  cmd.DrawMesh(RenderingUtils.fullscreenMesh, Matrix4x4.identity, workingMaterial);
  cmd.SetViewProjectionMatrices(sceneCamera.worldToCameraMatrix, sceneCamera.projectionMatrix);
  context.ExecuteCommandBuffer(cmd);
  CommandBufferPool.Release(cmd);
  context.Submit();`

but it didn't help and I can't figure out how to fix this / fixing this is beyond my skills.

**Comment by thestonefox** [comment]
This should be fixed now

**Pull request closed without merging**